stable_baselines/common/cmd_util.py

```diff
@@ -15,8 +15,63 @@
 from stable_baselines.common.vec_env import DummyVecEnv, SubprocVecEnv
 
 
+def make_vec_env(env_id, n_envs=1, seed=None, start_index=0,
+                 use_subprocess=False, start_method=None,
+                 monitor_path=None, wrapper_class=None, env_kwargs=None):
+    """
+    Create a wrapped, monitored VecEnv.
+
+    :param env_id: (str or gym.Env) the environment ID or the environment class
+    :param n_envs: (int) the number of environment you wish to have in parallel
+    :param seed: (int) the inital seed for random number generator
+    :param start_index: (int) start rank index
+    :param start_method: (str) method used to start the subprocesses.
+        See SubprocVecEnv doc for more information
+    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or `DummyVecEnv` when
+        `n_envs` > 1, `DummyVecEnv` is usually faster. Default: False
+    :param monitor_path: (str) Path to a folder where the monitor files will be saved.
+        If None, no file will be written, however, the env will still be wrapped
+        in a Monitor wrapper to provide additional information about training.
+    :param wrapper_class: (gym.Wrapper) Additional wrapper to use on the environment
```

Comment:
Could also be a function with single argument that wraps the environment in many things. This could be mentioned in the docs (current version implies you can only give a single Wrapper class).